packages/pcd/gpc-pcd/src/GPCPCDPackage.ts

```diff
@@ -251,7 +283,7 @@ export function getProveDisplayOptions(): ProveDisplayOptions<GPCPCDArgs> {
         argumentType: ArgumentTypeName.String,
         defaultVisible: true
       },
-      pod: {
+      [`${PODPCD_ARG_PREFIX}_pod0`]: {
```

Comment:
This isn't going to be sufficient if the args have different names.  What do you see if you try to prove here using a name other than podpcd_pod0?  The prove screen needs this information in order to give the user the right interface to select a PCD, and the validate function below is what filters them down to PODs.  Making the prove screen deal with argument names in a generic way is going to need more work.